### ISSUE TITLE
Option to hide app list/drawer.

### ISSUE THREAD

**Issue by sylvqx** [ottop/yam_launcher#19]
I really enjoy using your launcher! It’s clean and efficient. However, I’d love an option to hide the app list/drawer/menu for an even more minimalist experience.
This would enhance the customization options and align with the minimalist philosophy of the launcher, making it possible to just access the home screen.
Thank you for your amazing work.

**Comment by ottop** [comment]
Sounds reasonable. One separate task I have on my list of stuff to do is to maek the gestures customizable. I'll add in the option to disable any gesture as well, so you will be able to disable the app drawer this way.

**Comment by alrajdev** [comment]
If you are going for an even more minimalistic option you would definitely not have 10s of apps selected in your home screen, at most 5 would be the right amount for a minimalistic look, then if you disabled the App Drawer how would you access other apps which was not selected into your home screen?

**Comment by ottop** [comment]
> If you are going for an even more minimalistic option you would definitely not have 10s of apps selected in your home screen, at most 5 would be the right amount for a minimalistic look, then if you disabled the App Drawer how would you access other apps which was not selected into your home screen?

I would imagine that the idea of using it without the app drawer would be to open apps by switching the shortcut apps to new ones when needed.

**Comment by alrajdev** [comment]
Yeah I see that, but that would be cumbersome to do it every time. Cause some app are like predefined in the Home like 'Phone', 'SMS', 'Camera', then you would have to keep changing the Home menu just to open a different app.

Of course it is up to the user on how He/She uses the app, but we cannot call it a feature since the very purpose of a launcher app is to show the installed app to the user.

**Comment by ottop** [comment]
It's certainly not something that I would promote as a feature, but the implementation would simply be an ability for the user to do that as a part of entirely customizable gestures, so I think not forcing anything to be enabled is fine (of course the defaults would be as they are now with the essentials enabled).

**Comment by sylvqx** [comment]
> If you are going for an even more minimalistic option you would definitely not have 10s of apps selected in your home screen, at most 5 would be the right amount for a minimalistic look, then if you disabled the App Drawer how would you access other apps which was not selected into your home screen?

I suggested adding an extra feature to hide the app drawer as this would help users minimize screen time to zero, ideal for those seeking a more minimalist phone/dumb phone experience.
